modules/outputs.tf

```diff
@@ -0,0 +1,34 @@
+output "rcb_vm_name" {
+  value       = module.rcb_vm.name
```

Comment:
Treaba asta nu o sa mearga in terraform, daca ai nevoie sa folosesti un output in main.tf o sa scrii ceva de genul
`name = module.rcb_vm.name`
Nu e nevoie sa le definesi iar in root, cred ca poti sa stergi tot fisierul asta